docs/API.rst

```diff
@@ -64,11 +64,20 @@ error phase.
 
 .. _appinitialize-messages-loggingservice-overridehandlers-custom-:
 
-``App.initialize({ messages, loggingService, overrideHandlers, custom })``
+``App.initialize({ allowAnonymous, hydrateAuthenticatedUser, messages, loggingService, overrideHandlers, custom })``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``App.initialize`` method takes an options object with four possible
-keys:
+The ``App.initialize`` method takes an options object with the following possible keys:
+
+allowAnonymous
```

Comment:
Is this something that could be supplied as a prop to <AppProvider>? Does it make more sense on init or there I wonder. Will continue to think about as I read. Maybe:
```
<AppProvider allowAnonymous>

</AppProvider>
```